chef/cookbooks/bind9/recipes/default.rb

```diff
@@ -143,7 +146,8 @@ def make_zone(zone)
     group "root"
     notifies :reload, "service[bind9]"
     variables(zonefile_entries: zonefile_entries,
-              master_ip: master_ip)
+              master_ip: master_ip,
+              admin_addr: admin_addr)
```

Comment:
admin_addr seems to be undefined here - likely because we are in a function here while the other use of admin_addr is not.